LocalParks.API/Controllers/ApiPostcodesController.cs

```diff
@@ -0,0 +1,68 @@
+﻿using LocalParks.Core.Contracts;
+using LocalParks.Core.Models;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Threading.Tasks;
+
+namespace LocalParks.API.Controllers
+{
+    [ApiController]
+    [Authorize(AuthenticationSchemes = JwtBearerDefaults.AuthenticationScheme)]
```

Comment:
you can put authorise attribute on base class if all your controllers require authentication

Reply:
Ah ok, will do, thanks!